ts/features/wallet/bancomat/screen/BancomatDetailScreen.tsx

```diff
@@ -82,37 +85,50 @@ const BancomatDetailScreen: React.FunctionComponent<Props> = props => {
     caption: bancomat.abiInfo?.name ?? I18n.t("wallet.methods.bancomat.name")
   });
   return (
-    <DarkLayout
-      bounces={false}
-      title={I18n.t("wallet.methods.card.shortName")}
-      faqCategories={["wallet_methods"]}
-      allowGoBack={true}
-      topContent={<View style={styles.headerSpacer} />}
-      gradientHeader={true}
-      hideHeader={true}
-      contextualHelp={emptyContextualHelp}
-    >
-      <View style={styles.cardContainer}>
-        <BancomatCard enhancedBancomat={bancomat} />
-      </View>
-      <View spacer={true} extralarge={true} />
-      <View spacer={true} />
-
-      <View style={IOStyles.horizontalContentPadding}>
-        <BancomatInformation onAddPaymentMethod={() => startCoBadge(props)} />
-        <View spacer={true} />
-        <ItemSeparatorComponent noPadded={true} />
-        <View spacer={true} />
-        <PaymentMethodCapabilities paymentMethod={bancomat} />
-        <View spacer={true} />
-        <ItemSeparatorComponent noPadded={true} />
-        <View spacer={true} />
-        <UnsubscribeButton
-          onPress={() => present(() => props.deleteWallet(bancomat.idWallet))}
+    <>
+      {props.isLoadingDelete ? (
```

Comment:
There is a reason why this fragment is used? Can we remove it in your opinion?
```suggestion
    props.isLoadingDelete ? (
```
If for you make sense the same is in the other payment methods screen.